ocs_ci/ocs/ui/views.py

```diff
@@ -252,6 +252,23 @@
     "test-pvc-rbd": ("a[title='test-pvc-rbd']", By.CSS_SELECTOR),
     "Block": ("input[value='Block']", By.CSS_SELECTOR),
     "Filesystem": ("input[value='Filesystem']", By.CSS_SELECTOR),
+    "search-project": ("input[placeholder='Select Project...']", By.CSS_SELECTOR),
+    "expand_pvc": ("button[data-test-action='Expand PVC']", By.CSS_SELECTOR),
+    "resize-value": ("//input[@name='requestSizeValue']", By.XPATH),
+    "expand-btn": ("#confirm-action", By.CSS_SELECTOR),
+    "pvc-status": (
+        "dd[data-test-id='pvc-status'] span[data-test='status-text']",
+        By.CSS_SELECTOR,
+    ),
+    "test-project-link": ("//a[normalize-space()='{}']", By.XPATH),
+    "expected-capacity": (
+        "//dd[contains(text(),'{}') and @data-test='pvc-requested-capacity']",
+        By.XPATH,
```

Comment:
You can use this locator.
``` "expected-capacity": (('//*[@data-test="pvc-requested-capacity"]', By.XPATH) ```

Reply:
This locator will not work in my test scenario and will return True even if the 'Capacity' value doesn't change after PVC resize.

Reply:
ok